uploader-test.js

```diff
@@ -0,0 +1,43 @@
+/**
+This is not a proper test. Only used as proof of concept.
```

Comment:
Please DO NOT merge this test. I left it here as a reference. Once everything look OK on index.js, I'll remove this file before the final merge :)